kotlintest-extensions/src/main/kotlin/io/kotlintest/extensions/locale/TimezoneExtension.kt

```diff
@@ -1,22 +1,10 @@
-package io.kotlintest.extensions
+package io.kotlintest.extensions.locale
```

Comment:
I'm not sure we need a package for a single file

Reply:
There are two classes, so I created a package to hold 2 files

Reply:
I'm afraid that we'll clog the extensions root package with too many things and then we'll start moving things to their own package later.

The same thing happened in KT-assertions, and I'm trying to avoid this happening here too

Reply:
Yes fair enough.